assets/src/edit-story/components/helpCenter/useHelpCenter/test/effects.js

```diff
@@ -0,0 +1,204 @@
+    expect(isNextDisabled).toBe(false);
+  });
+
+  it('sets isLeftToRightTransition to true if navigationIndex is increasing', () => {
+    const previousState = mockState({
+      navigationIndex: 0,
+      navigationFlow: ['tip_1', 'tip_2'],
+    });
+    const nextState = mockState({
+      navigationIndex: 1,
+      navigationFlow: ['tip_1', 'tip_2'],
+    });
+    const { isLeftToRightTransition } = deriveState(previousState, nextState);
+    expect(isLeftToRightTransition).toBe(true);
+  });
+
+  it('sets isLeftToRightTransition to false if navigationIndex is decreasing', () => {
+    const previousState = mockState({
+      navigationIndex: 2,
+      navigationFlow: ['tip_1', 'tip_2'],
+    });
+    const nextState = mockState({
+      navigationIndex: 1,
+      navigationFlow: ['tip_1', 'tip_2'],
+    });
+    const { isLeftToRightTransition } = deriveState(previousState, nextState);
+    expect(isLeftToRightTransition).toBe(false);
+  });
+
+  it('sets hasBottomNavigation to true if navigationIndex is not on menu index', () => {
+    const previousState = mockState({
+      navigationIndex: -1,
+    });
+    const nextState = mockState({
+      navigationIndex: 0,
+    });
+    const { hasBottomNavigation } = deriveState(previousState, nextState);
+    expect(hasBottomNavigation).toBe(true);
+  });
+
+  it('sets hasBottomNavigation to false if navigationIndex is on menu index', () => {
+    const previousState = mockState({
+      navigationIndex: 4,
+    });
+    const nextState = mockState({
+      navigationIndex: -1,
+    });
+    const { hasBottomNavigation } = deriveState(previousState, nextState);
+    expect(hasBottomNavigation).toBe(false);
+  });
+
+  it('sets a tip to read if the navigation index is not on the menu or done index', () => {
+    const navigationFlow = Object.keys(TIPS);
+    const previousState = mockState({
+      navigationIndex: -1,
+      navigationFlow: navigationFlow,
+    });
+    const nextState = mockState({
+      navigationIndex: 0,
+      navigationFlow: navigationFlow,
+    });
+    const { readTips } = deriveState(previousState, nextState);
+    expect(readTips[navigationFlow[0]]).toBe(true);
+  });
+
+  it('resets the navigation index to the main menu when opening', () => {
+    const previousState = mockState({
+      isOpen: false,
+      navigationIndex: 4,
+    });
+    const nextState = mockState({
+      isOpen: true,
+      navigationIndex: 4,
+    });
+    const { navigationIndex } = deriveState(previousState, nextState);
+    expect(navigationIndex).toBe(-1);
+  });
+
+  it('retains navigation index when staying open between state transitions', () => {
+    const previousState = mockState({
+      isOpen: true,
+      navigationIndex: 4,
+    });
+    const nextState = mockState({
+      isOpen: true,
+      navigationIndex: 4,
+    });
+    const { navigationIndex } = deriveState(previousState, nextState);
+    expect(navigationIndex).toBe(4);
+  });
+});
```

Comment:
This is all old code moved from the other test file into here. since it's all testing effects I moved it in here and annotated todo tasks to split it up into atomic effect tests